fontforge/python.c

```diff
@@ -1751,7 +1753,20 @@ return( reto );
 /* Points */
 /* ************************************************************************** */
 
-static PyFF_Point *PyFFPoint_CNew(double x, double y, int on_curve, int sel, char *name) {
+static const char *py_point_types[] = { "splineCorner", "splineCurve", "splineHVCurve",
+				     "splineTangent", NULL };
```

Comment:
By making `splineCorner` zero the field becomes compatible with the `smooth` boolean used in some spline formats, including glif. If you set the field to zero or one according to that flag, and then convert with `pconvert_flag_smooth`, you get the same result as you would importing a UFO file.